javascript/javascript3/week1/Weather/script.js

```diff
@@ -0,0 +1,44 @@
+//Weather API call
+const APIKEY = "9469946e53df5e8174588db17f99cd64";
+const limit = 3;
+
+const country = document.getElementById('search-country');
+const cityName = document.getElementById('search-city');
+const button = document.getElementById('button');
+
+const resultUl = document.getElementById('result-ul');
+
+button.addEventListener('click', (res)=>{
+    res.preventDefault();
+    let weatherCall = new Promise((resolve,reject) =>{
```

Comment:
Nitpick(naming): Not sure how I would name this but call feels confusing! 
Maybe just `promiseForWeather`? :D 

But I think we can get rid of it all together, see my comment below!